<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ProfileWidget</class>
    <widget class="QWidget" name="ProfileWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>User Profile</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">font-size: 16pt; font-weight: bold;</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="usernameLabel">
                    <property name="text">
                        <string>Username: </string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="fullNameLabel">
                    <property name="text">
                        <string>Full Name: </string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="roleLabel">
                    <property name="text">
                        <string>Role: </string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPushButton" name="changePasswordButton">
                    <property name="text">
                        <string>Change Password</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QGroupBox" name="passwordGroup">
                    <property name="title">
                        <string>Change Password</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="label_4">
                                <property name="text">
                                    <string>New Password:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="newPassword">
                                <property name="echoMode">
                                    <enum>QLineEdit::Password</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="label_5">
                                <property name="text">
                                    <string>Confirm Password:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="confirmPassword">
                                <property name="echoMode">
                                    <enum>QLineEdit::Password</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0" colspan="2">
                            <widget class="QPushButton" name="savePasswordButton">
                                <property name="text">
                                    <string>Save Password</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>40</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QPushButton" name="logoutButton">
                    <property name="text">
                        <string>Logout</string>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">background-color: #f44336; color: white; font-weight: bold;</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
